openshift/tests-extension/test/webhooks.go

```diff
@@ -227,30 +231,30 @@ var _ = Describe("[sig-olmv1][OCPFeatureGate:NewOLMWebhookProviderOpenshiftServi
 
 			Eventually(func(g Gomega) {
 				resourceName := fmt.Sprintf("tls-deletion-test-%s", rand.String(5))
-				resource := newWebhookTestV1(resourceName, webhookOperatorInstallNamespace, true)
+				resource := newWebhookTest(resourceName, webhookOperatorInstallNamespace, true)
 
-				_, err := dynamicClient.Resource(webhookTestGVRV1).Namespace(webhookOperatorInstallNamespace).Create(ctx, resource, metav1.CreateOptions{})
+				_, err := dynamicClient.Resource(webhookTestV1).Namespace(webhookOperatorInstallNamespace).Create(ctx, resource, metav1.CreateOptions{})
 				g.Expect(err).ToNot(HaveOccurred(), fmt.Sprintf("failed to create test resource %s: %v", resourceName, err))
 
-				err = dynamicClient.Resource(webhookTestGVRV1).Namespace(webhookOperatorInstallNamespace).Delete(ctx, resource.GetName(), metav1.DeleteOptions{})
+				err = dynamicClient.Resource(webhookTestV1).Namespace(webhookOperatorInstallNamespace).Delete(ctx, resource.GetName(), metav1.DeleteOptions{})
 				g.Expect(client.IgnoreNotFound(err)).ToNot(HaveOccurred(), fmt.Sprintf("failed to delete test resource %s: %v", resourceName, err))
 			}).WithTimeout(2 * time.Minute).WithPolling(10 * time.Second).Should(Succeed())
 		})
 	})
 
-var webhookTestGVRV1 = schema.GroupVersionResource{
+var webhookTestV1 = schema.GroupVersionResource{
 	Group:    "webhook.operators.coreos.io",
 	Version:  "v1",
 	Resource: "webhooktests",
 }
 
-var webhookTestGVRV2 = schema.GroupVersionResource{
+var webhookTestV2 = schema.GroupVersionResource{
 	Group:    "webhook.operators.coreos.io",
 	Version:  "v2",
 	Resource: "webhooktests",
 }
 
-func newWebhookTestV1(name, namespace string, valid bool) *unstructured.Unstructured {
+func newWebhookTest(name, namespace string, valid bool) *unstructured.Unstructured {
```

Comment:
Again, this is not anything major at all, it was just bothering me that this function returns a webhook, so that we can use that in our tests, so `newTestWebhook` seems more of an appropriate name, than `newWebhookTest`. `newWebhookTest` means this is a test itself, which it is not.

Reply:
we are asking for a NEW struct of `WebhookTest `

Reply:
See your orginal test: https://github.com/openshift/origin/pull/30059/files#diff-6dd6fa78ac85235012d3c9910f8510bc1640c830a83888681bd5922cec0dffcbR332-R341